test/tsd.ts

```diff
@@ -837,7 +837,16 @@ function testHiveStores() {
             promiseListOfJSONValue = Backendless.Hive(str).ListStore(str).get(num, num);
 
             promiseNumber = Backendless.Hive(str).ListStore(str).set([str, str]);
```

Comment:
```
promiseNumber = Backendless.Hive(str).ListStore(str).set(str);
promiseNumber = Backendless.Hive(str).ListStore(str).set(num);
...
```

Reply:
It is not valid.
If only one arg, it should be only array

Reply:
ok